images/system-container.iuml

```diff
@@ -11,25 +11,44 @@ LAYOUT_AS_SKETCH()
 title Social Care System Container Diagram
 
 Person(care_practitioners, Social Care Practitioner, "<$user> <$user>\n Adult and Children" )
-Person_Ext(data_insights_team, Data and Insights Team, "<$user>" )
+Person(data_insight_team, Data and Insight Team )
 
-System_Ext(google_auth, "Google Groups", "Provides User Authentication and Authorisation")
-System_Ext(google_forms, "Google Forms", "Temporary data submissions (partial?)")
-System_Ext(qlik, "Qlik Analytics", "Case progress reporting")
+System_Ext(qlik, "Qlik Analytics", "ETL & Case progress reporting")
+
+System_Boundary(googles, "Google") {
+  System_Ext(google_auth, "Google Groups", "Provides User Authentication and Authorisation")
+  System_Ext(google_forms, "Google Forms", "Data submissions (57 forms)")
+  System_Ext(google_sheets, "Google Sheets", "Form submission data")
+  System_Ext(google_docs, "Google Docs", "Generated for viewing (AppScript)")
+}
 
 System_Boundary(system, "Social Care") {
 
-    Container(social_care_front_end, "Social Care Front End", "Lambda, React - TypeScript", "Allows practitioners to edit case information.")
-    Container(social_care_service_api, "Social Care Case Viewer Service API", "Lambda, .Net - C#", "Provides some abstraction")
-    Container(mosaic_historic_data_api, "Residents Social Care Platform API",  "Lambda, .Net - C#", "Serves historic read-only case and visit data")
+  Container(social_care_front_end, "Social Care Front End", "Lambda, React - TypeScript", "Allows practitioners to edit case information.")
```

Comment:
```suggestion
  Container(social_care_front_end, "Social Care Front End", "Lambda, Next.js (React) - TypeScript", "Allows practitioners to edit case information.")
```
I would add `Next.js` that is the framework that we are using, as it's with next that we proxy the APIs.

Also, I don't know if it's worth adding it, but we are already giving the practitioners the ability to add people to the system and in this sprint, we should (hopefully) be able to edit them.

Reply:
I've added `Next.js` as per you suggestion - thanks :)

I'm not convinced the specific functionalities are useful at this level - so won't add it for now.